src/components/annotations/draw_arrow_head.js

```diff
@@ -127,7 +127,7 @@ module.exports = function drawArrowHead(el3, ends, options) {
 
         d3.select(el.parentNode).append('path')
             .attr({
-                'class': el3.attr('class'),
+                class: el3.attr('class'),
```

Comment:
@alexcjohnson Do you recall if quotes around `class` are needed here?

Reply:
`class` is a reserved word, so it made me squeamish to leave it unquoted. But [This note](https://mathiasbynens.be/notes/javascript-properties) implies it's been fine for quite a long time:

> Note: ECMAScript 3 didn’t allow the use of unquoted reserved words as property names. Here’s the full list of ES3 reserved words: abstract, boolean, break, byte, case, catch, char, class, const, continue, debugger, default, delete, do, double, else, enum, export, extends, false, final, finally, float, for, function, goto, if, implements, import, in, instanceof, int, interface, long, native, new, null, package, private, protected, public, return, short, static, super, switch, synchronized, this, throw, throws, transient, true, try, typeof, var, void, volatile, while, and with. Avoid using these as unquoted property names if backwards compatibility is a concern.

This is [fine in ES5](https://www.w3schools.com/js/js_es5.asp#mark_reserved) and beyond 🎉